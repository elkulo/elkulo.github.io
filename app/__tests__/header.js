import React from "react"
import renderer from "react-test-renderer"
import Test from "../src/components/header/header"

describe("Test", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Test />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
