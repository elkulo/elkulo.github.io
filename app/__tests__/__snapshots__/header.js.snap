// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test renders correctly 1`] = `
<header
  className="site-header site-header--sidebar"
>
  <div
    className="wrap"
  >
    <div
      className="site-header__container"
    >
      <div
        className="site-header__container--primary"
      >
        <div
          className="site-branding"
        >
          <p
            className="site-branding__title site-title"
          >
            <a
              href="/"
            >
              My Title
            </a>
          </p>
        </div>
      </div>
    </div>
  </div>
</header>
`;
